src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionLocationService.java

```diff
@@ -85,6 +94,29 @@ public String deleteSubscriptionByLocation(String locationId, String provenanceU
 
     }
 
+    private void deleteAllSubscriptionListTypeForLocation(List<Subscription> locationSubscriptions) {
+        List<String> uniqueUsers = locationSubscriptions.stream()
+            .map(Subscription::getUserId).distinct().toList();
+        List<UUID> subscriptionListTypesIds = new ArrayList<>();
+        int minNoOfSubscriptions = 1;
```

Comment:
Not sure it's worth pulling this out into a variable as it's only used once on line 107.

Reply:
fixed.